src/PantosHubStorage.sol

```diff
@@ -19,8 +19,7 @@ struct PantosHubStorage {
     uint256 numberActiveBlockchains;
     uint256 currentBlockchainId;
     mapping(uint256 => PantosTypes.BlockchainRecord) blockchainRecords;
-    uint256 minimumTokenStake;
```

Comment:
Have you tested the removal of this state variable after doing an update?

The removal of minimumTokenStake causes that all following state variables are moving up, which could lead to some problems if those values are not updated after doing a contract update, since we will read from the "wrong" slot.

Reply:
We have to redeploy after these changes.